src/types/line.js

```diff
@@ -88,14 +88,16 @@ export default class LineAnnotation extends Element {
 				y = min;
 				y2 = max;
 			}
+			_horizontal = !scale.isHorizontal();
 		}
 		return {
 			x,
 			y,
 			x2,
 			y2,
 			width: x2 - x,
-			height: y2 - y
+			height: y2 - y,
+			_horizontal
```

Comment:
`_horizontal: !scale.isHorisontal()`

Reply:
`scale` could be `undefined` (see previous if statement) and therefore I'll get an error, with your code.

Reply:
Ah, right!